fvm/fvm_test.go

```diff
@@ -3229,3 +3231,152 @@ func TestAccountCapabilitiesPublishEntitledRejection(t *testing.T) {
 		}),
 	)
 }
+
+func TestCrypto(t *testing.T) {
+	t.Parallel()
+
+	chain, vm := createChainAndVm(flow.Testnet)
+
+	ctx := fvm.NewContext(
+		fvm.WithChain(chain),
+		fvm.WithCadenceLogging(true),
+	)
+
+	script := []byte(`
+      import Crypto
```

Comment:
Could you please also add a tests for importing the `Crypto` contract using the explicit address

Reply:
Good idea! Added in 389cc0eea1b736c034261d484e74bf2e9b22b3f7